account_invoice_inter_company/models/account_move.py

```diff
@@ -23,6 +23,12 @@ class AccountMove(models.Model):
         copy=False,
         prefetch=False,
     )
+    related_bill_id = fields.Many2one(
```

Comment:
This one is not filled for the past ones. Why not using the inverse link?

Reply:
@pedrobaeza I dont know how to use inverse here. related_bill_id  can be computed using search by auto_invoice_id. Is that what you mean ?

Reply:
@ilyasProgrammer Done ?